app/uk/gov/hmrc/rasapi/config/microserviceWiring.scala

```diff
@@ -35,19 +39,33 @@ trait WSHttp extends
   HttpPatch with WSPatch with AppName {
   override val hooks: Seq[HttpHook] = NoneRequired
 
+  override protected def actorSystem: ActorSystem = akka.actor.ActorSystem()
```

Comment:
Change to use the Play.current as this will load the actor system rather than potentially creating one